specutils/spectra/xrayspectrum1d.py

```diff
@@ -0,0 +1,573 @@
+import logging
+
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+from .spectrum1d import Spectrum1D
+
+__all__ = ['XraySpectrum1D', 'AreaResponse', 'ResponseMatrix']
+
+# For dealing with varied unit string choices
+EV   = ['eV', 'ev']
+KEV  = ['kev', 'keV']
+ANGS = ['angs', 'Angs', 'Angstrom', 'angstrom', 'Angstroms', 'angstroms', 'A', 'a']
+
+## An introduction to X-ray analysis can be found here:
+## http://cxc.cfa.harvard.edu/xrayschool/talks/intro_xray_analysis.pdf
+
+def _unit_parser(unit_string):
+    """
+    A function for matching unit strings to the correct Astropy unit
+    """
+    if unit_string in EV:
+        return u.eV
+    if unit_string in KEV:
+        return u.keV
+    if unit_string in ANGS:
+        return u.angstrom
+
+class XraySpectrum1D(Spectrum1D):
+    """
+    Spectrum properties specific to X-ray data
+
```

Comment:
Really: Binned counts spectra. Could be a Gamma ray spectrum... but that's just a side-note.